code/modules/food&drinks/food/snacks_burgers.dm

```diff
@@ -18,6 +18,26 @@
 	desc = "A bloody burger."
 	bonus_reagents = list("vitamin" = 4)
 
+/obj/item/weapon/reagent_containers/food/snacks/burger/human/lizard
+	name = "lizard burger"
+	desc = "Made from non-valid tailmeat."
+	bonus_reagents = list("vitamin" = 4)
+
+/obj/item/weapon/reagent_containers/food/snacks/burger/human/bird
+	name = "bird burger"
+	desc = "Tastes like chicken."
+	bonus_reagents = list("vitamin" = 4)
+
+/obj/item/weapon/reagent_containers/food/snacks/burger/human/robo
```

Comment:
There's already one roboburger ingame, and i believe its name is roburger, but that shit has nanomachines. I'd suggest checking its behaviour and using that one instead, since it has no recipe currently most likely. But check if it spawns with nanites...if it does, it may be a bit too bad